packages/divider/stories/divider.stories.ts

```diff
@@ -10,39 +10,31 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import { html } from '@open-wc/demoing-storybook';
-import { TemplateResult } from '@spectrum-web-components/base';
+import { html, TemplateResult } from '@spectrum-web-components/base';
 
-import '../sp-rule.js';
-import './typography-decorator.js';
+import '../sp-divider.js';
 import '@spectrum-web-components/action-button/sp-action-button.js';
 import '@spectrum-web-components/icon/sp-icon.js';
-import {
-    AlignLeftIcon,
-    AlignRightIcon,
-} from '@spectrum-web-components/icons-workflow';
+import '@spectrum-web-components/icons-workflow/icons/sp-icon-align-left.js';
+import '@spectrum-web-components/icons-workflow/icons/sp-icon-align-right.js';
 
 export default {
-    title: 'Rule',
-    decorators: [
-        (story: () => TemplateResult): TemplateResult => html`
-            <typography-decorator .story=${story()}></typography-decorator>
```

Comment:
This related to the `sideEffects` listing note above and the ability for the regressions to match.